TombEngine/Game/savegame.cpp

```diff
@@ -825,10 +823,11 @@ const std::vector<byte> SaveGame::Build()
 		Save::ItemBuilder serializedItem{ fbb };
 
 		if (Objects.CheckID(itemToSerialize.ObjectNumber, true))
-			serializedItem.add_anim_number(itemToSerialize.Animation.AnimNumber - Objects[itemToSerialize.ObjectNumber].animIndex);
+			serializedItem.add_anim_number(itemToSerialize.Animation.AnimNumber);
 
 		serializedItem.add_next_item(itemToSerialize.NextItem);
 		serializedItem.add_next_item_active(itemToSerialize.NextActive);
+		serializedItem.add_anim_number(itemToSerialize.Animation.AnimNumber);
```

Comment:
This causes an exception while saving the game. We are adding the same field twice. I suggest to remove the similar code above (along with `if (Objects.CheckID(itemToSerialize.ObjectNumber, true))` condition).